pkg/locale/localizedata/ja.yaml

```diff
@@ -32,3 +36,27 @@ PeroidOutOfRange: "期間は過去30日以内を選択してください。"
 
 # environment error
 ProjectDisabled: "projectのデータが無効化されています"
+
+#############################
+# Domain Events
+#############################
+
+UnknownOperation: "不明な操作が実行されました"
+Created: "{{ .Field_1 }}が作成されました"
+Enabled: "{{ .Field_1 }}が有効化されました"
+Disabled: "{{ .Field_1 }}が無効化されました"
+Archived: "{{ .Field_1 }}がアーカイブされました"
+Unarchived: "{{ .Field_1 }}がアーカイブから解除されました"
+Deleted: "{{ .Field_1 }}が削除されました"
+Added: "{{ .Field_1 }}が追加されました"
+Changed: "{{ .Field_1 }}が変更されました"
+Updated: "{{ .Field_1 }}が更新されました"
+Started: "{{ .Field_1 }}が開始されました"
+Stopped: "{{ .Field_1 }}が停止されました"
+Finished: "{{ .Field_1 }}が終了されました"
+Uploaded: "{{ .Field_1 }}がアップロードされました"
+Cloned: "{{ .Field_1 }}がクローンされました"
+Reset: "{{ .Field_1 }}がリセットされました"
+TrialConverted: "Trialを正式なProjectに変換しました"
+DelayableSet: "feature flagを初回リクエスト時にキューに入れるように変更されました"
```

Comment:
We don't use the `DelayableSet` and `UndelayableSet` anymore. You can ignore it.

Reply:
@cre8ivejp 

Can I remove `FEATURE_EVALUATION_DELAYABLE_SET` and `FEATURE_EVALUATION_UNDELAYABLE_SET` field in proto/event/domain/event.proto? Otherwise, `TestImplementedLocalizedMessage` will be failed since we don't define `DelayableSet` and `UndelayableSet`.

Reply:
Yes. Thanks.